### ISSUE TITLE
Question: Is there a way to center the modal on the screen?

### ISSUE THREAD

**Issue by pedrogarciyalopez** [AstrOOnauta/react-native-international-phone-number#34]
Hello. thank you for awesome lib!
Is there a way to center the modal on the screen? How can i do that?

**Comment by AstrOOnauta** [comment]
@pedrogarciyalopez unfortunately not...

The countries modal works as a bottom sheet only (or full page setting the `height` props to `100%`). Its the actual  behavior, sorry..

🚀 AstrOOnauta 🚀

**Comment by pedrogarciyalopez** [comment]
@AstrOOnauta thank you for quick reply.

If someone needs to do the same thing, I found a simple solution. I just set the margin: 'auto' in the styles of the modal window. We need this because we are using react-native-web, and on the desktop, all our modal windows are centered on the screen.

```
modal: {
  padding: 25,
  paddingBottom: 0,
  borderRadius: 15,
  margin: 'auto',
}
```
<img width="805" alt="image" src="https://github.com/AstrOOnauta/react-native-international-phone-number/assets/14232660/62d35d7d-9b8e-4df8-b226-578f658277d1">

**Comment by pedrogarciyalopez** [comment]
@AstrOOnauta thank you for quick reply.

If someone needs to do the same thing, I found a simple solution. I just set the margin: 'auto' in the styles of the modal window. We need this because we are using react-native-web, and on the desktop, all our modal windows are centered on the screen.

```
modal: {
    padding: 25,
    paddingBottom: 0,
    borderRadius: 15,
    margin: 'auto',
  }
```
<img width="802" alt="image" src="https://github.com/AstrOOnauta/react-native-international-phone-number/assets/14232660/0dd74779-eb7a-41ee-b97a-073226856154">
